2024/wg-fedid.html

```diff
@@ -246,7 +246,19 @@ <h3>
             <p class="milestone"><b>Expected completion:</b> CR in Q1 2025</p>
           </dd>
         </dl>
-
+        <p>
+          Depending on the incubation progress, interest from multiple implementers, and the
+          consensus of the Group participants, the Group may also produce a Recommendation-track
+          specification for the following document:
+        </p>
+        <dl>
+          <dt><a href="https://wicg.github.io/digital-identities/">Digital Credentials</a></dt>
+          <dd>This specification, incubated in <a href="https://www.w3.org/groups/cg/wicg">Web
+            Platform Incubator Community Group</a>, specifies an API to enable user agents to
+            mediate access to, and presentation of, digital credentials such as a driver's
```

Comment:
I suggest to include issuance as well. Wallet based applications are incomplete without the issuance part. Furthermore, providing the web with an interoperable way for issuance will facilitate user's choice of their wallet.